src/utils/fundWallet.js

```diff
@@ -0,0 +1,94 @@
+const { PrivateKey } = require('@dashevo/dashcore-lib');
+
+const EVENTS = require('../EVENTS');
+
+/**
+ *
+ * @param {Account} walletAccount
+ * @param {string} id - transaction id
+ * @return {Promise<string>}
+ */
+function waitForTransaction(walletAccount, id) {
+  return new Promise(((resolve) => {
+    const listener = (event) => {
+      const { payload: { transaction } } = event;
+
+      if (transaction.id === id) {
+        walletAccount.removeListener(EVENTS.FETCHED_CONFIRMED_TRANSACTION, listener);
+
+        resolve(transaction.id);
+      }
+    };
+
+    walletAccount.on(EVENTS.FETCHED_CONFIRMED_TRANSACTION, listener);
+  }));
+}
+
+/**
+ *
+ * @param {Wallet} faucetWallet
+ * @param {Wallet} recipientWallet
+ * @param {number} amount
+ * @param {Object} [options]
+ * @param {boolean} [options.mineBlock=false]
+ * @param {boolean} [options.isRegtest=false]
+ * @return {Promise<void>}
+ */
+async function fundWallet(faucetWallet, recipientWallet, amount, options = {}) {
+  // eslint-disable-next-line no-param-reassign
+  options = {
+    mineBlock: false,
+    ...options,
+  };
+
+  if (options.mineBlock) {
+    const privateKey = new PrivateKey();
+
+    await faucetWallet.transport.client.core.generateToAddress(
+      1,
+      privateKey.toAddress(faucetWallet.network).toString(),
+    );
+  }
+
+  if (options.isRegtest) {
+    const initialHeight = await faucetWallet.transport.client.core.getBestBlockHeight();
```

Comment:
Shouldn't it be in mn-bootstrap where we generate dash to address in order to use it faucet?